jme3-core/src/main/java/com/jme3/renderer/Camera.java

```diff
@@ -1480,13 +1555,17 @@ public Vector3f getScreenCoordinates(Vector3f worldPosition, Vector3f store) {
     }
 
     /**
+     * Determines the display width.
+     *
      * @return the width/resolution of the display.
      */
     public int getWidth() {
         return width;
     }
 
     /**
+     * Determines the display height.
+     *
      * @return the height/resolution of the display.
```

Comment:
And this too.

Reply:
The choice between "Gets" and "Determines" isn't addressed in the style guide.